lib/fluent/plugin/filter_process_redfishalert.rb

```diff
@@ -56,13 +59,21 @@ def callRedfishGetAPI(host, resourceURI)
       end
     end
 
+    #we are using nodeID as the unique identifier for dell iDRAC
+    #also, for dell nodeID=SKU=ChassisServiceTag but differs from SN
     def getRMCSerialNumber(host)
```

Comment:
if this is different from the serial number, maybe instead of adding the logic to getRMCSerialNummber we can write a new method like "getMachineIdentifier" and  call that or this method depending on the hardwaretype. If we're not getting the RMC serial number here we should write a new method for this

Reply:
Yea, From our investigation some services downstream  already use node id as the unique identifier for dell, we decided to continue with this format.